package.json

```diff
@@ -20,7 +20,8 @@
     "create-react-component": "yo react-domain"
   },
   "dependencies": {
-    "@theforeman/vendor": "^4.3.0",
+    "@theforeman/vendor": "<4.9.0",
+    "@theforeman/vendor-core": "<4.9.0",
```

Comment:
I added this because vendor has the vendor-core version described as "^4.8.0" which pulls in 4.10.0 ..